avocado/core/utils/messages.py

```diff
@@ -2,7 +2,7 @@
 import sys
 import time
 
-from avocado.core.streams import BUILTIN_STREAMS
+from avocado.core.output import split_loggers_and_levels
```

Comment:
There's no `split_loggers_and_levels` in `avocado.core.output` at this point.  This breaks `avocado-runner-avocado-instrumented` and possibly others.